### PR TITLE
Let homescreen render robots.txt

### PR DESCRIPTION
**Pull request by jonasheinrich**
Currently we have no robots.txt. We should change that.

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://jonasheinrich-fix-robots-txt.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://jonasheinrich-fix-robots-txt.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

**Comment by ulferts** [comment]
The implementation looks solid. 

However we noticed the problem by having a live issue on community.openproject.org. So I guess it would be really good if we add a feature spec to ensure, that we notice problems faster the next time around.

File: app/views/homescreen/robots.text.erb [hunk 1]
```diff
@@ -28,9 +28,9 @@ See doc/COPYRIGHT.rdoc for more details.
 ++#%>
 User-agent: *
 <% @projects.each do |p| -%>
-  Disallow: /projects/<%= p.to_param %>/repository
-Disallow: /projects/<%= p.to_param %>/issues
+Disallow: /projects/<%= p.to_param %>/repository
```
Comment (why-el) [inline app/views/homescreen/robots.text.erb:31]: why not use a path helper here?

**Comment by oliverguenther** [comment]
As @jonasheinrich is on vacation for the time being, I'm taking over the changes over at https://github.com/opf/openproject/pull/3611.

**Pull request closed without merging**